nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-client-dto/src/main/java/org/apache/nifi/web/api/entity/ProcessGroupEntity.java

```diff
@@ -56,6 +56,9 @@ public class ProcessGroupEntity extends ComponentEntity implements Permissible<P
 
     private ParameterContextReferenceEntity parameterContext;
 
+    private Boolean logToOwnFile;
+    private String logFileSuffix;
```

Comment:
Is there a reason these are necessary at the Entity level? Having them at the DTO level should be sufficient.